src/runtime/workspace_pool.cc

```diff
@@ -128,6 +128,8 @@ WorkspacePool::~WorkspacePool() {
    * Comment out the destruct of WorkspacePool, due to Segmentation fault with
    * MXNet Since this will be only called at the termination of process, not
    * manually wiping out should not cause problems.
+   * Note, this will cause memory leak without the following code, so, maybe
```

Comment:
Since we've stopped support MXNet and already close MXNet unit testing, can you try un-comment the code, and see whether it eliminates the memory leak?

Reply:
I'm not familiar with this code, maybe you can have a try or I can test it later